rundown/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import (
+    RundownCreateView,
+    RundownUpdateView
+)
+
+urlpatterns = [
+    path('', RundownCreateView.as_view(), name='rundown-create'),
+    path('update/<uuid:id>', RundownUpdateView.as_view(), name='rundown-update'),
```

Comment:
Because there would be a delete endpoint as well and you are using APIView, I think update here should be removed and be left like this:

path('<uuid:id>', RundownUpdateView.as_view(), name='rundown-update'),